Libs/Common/ShapeworksUtils.h

```diff
@@ -22,6 +22,12 @@ class ShapeworksUtils
   /// return elapsed time in seconds, optionally printing to console
   static double elapsed(time_point start, time_point end, bool print_elapsed = true);
 
+  /// returns random seed
```

Comment:
We probably want to replace this with the random number generator itself, since as written, it will use the same seed each time functions like `FECVDDecimationModifier::Initialize` are called.

The reasons for this in the first place are debugging and testing, which should perhaps also be documented. 

The downside to replacing this with the rng itself is that threaded calls to these functions will still not always be reproducible. Do we need both [for testing/debugging]?

Reply:
I do think we need it for testing purposes

Reply:
Done